File: Old JAVA/Enumerise/ConditionEnum.java
package Enumerise;

public enum ConditionEnum {

MORE(">",""),
LESS("<",""),
INCLUDE("INCLUDE",""),
EXCLIDE("EXCLUDE","")

			;
			


		private String name = "";
		private String unite = "";
		private int coef = 0;
		private String Label="";

		//Constructeur
	ConditionEnum(String name, String unit ){
		this.name = name;
this.unite=unit;
		}




		public String nameToString(){
		return name;
		}
		public String getName(){
		return name;
		}
}

File: composeApp/src/commonMain/kotlin/fr/vetbrain/vetnutri_mp/Enumerise/ConditionEnum.kt
package fr.vetbrain.vetnutri_mp.Enumerise

import fr.vetbrain.vetnutri_mp.Data.Labelable

enum class ConditionEnum( override val label: String,  val unite: String) :Labelable{
    MORE(">", ""),
    LESS("<", ""),
    INCLUDE("INCLUDE", ""),
    EXCLUDE("EXCLUDE", "");

  
}
